source/change-streams.rst

```diff
@@ -224,11 +228,32 @@ Driver API
 
   interface Collection {
     /**
+     * @since 3.6
      * @returns a change stream on a specific collection.
      */
     watch(pipeline: Document[], options: Optional<ChangeStreamOptions>): ChangeStream;
   }
 
+  interface Database {
+    /**
+     * Allows a client to observe all changes in a database
+     * @returns a change stream  on all collections in a database
+     * @since 4.0
+     * @note excludes system collections
```

Comment:
Would it make sense to combine this with the description?

> Allows a client to observe changes across all collections within a database, excluding system collections.

Then, I'd replace this `@note` with:

```
@see https://docs.mongodb.com/manual/reference/system-collections/
```

Reply:
done